components/containers/Light.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Does `import { ReactNode } from  "react";` work? If so, I think it's better to go with this.

Reply:
Yes that works!